example_simple_classification.py

```diff
@@ -41,7 +41,7 @@ def main():
     # Run 4 built in algorithms to see which one looks best at first
     hcai.ensemble()
 
-    hcai.get_advanced_features().advanced_random_forest_classifier()
+    hcai.get_advanced_features().random_forest_classifier()
 
```

Comment:
Next week, would love a tour of where get_advanced_features is headed.

Sounded like CAFE was fine working with linear, rf, and gradient boosting algorithms to start--hopefully that's still the case. Appreciate your extensive work on this!